### ISSUE TITLE
Symbol rate of encoder and decoder should match

### ISSUE THREAD

**Issue by jaakjensen** [float32/qpsk#1]
READ ME says "sample_rate and symbol_rate are measured in Hz and should approximate the rates of the incoming signal (they need not match the rates of the encoded audio file). sample_rate must be 6, 8, 12, or 16 times the symbol_rate."

I've found that mis-matching sample rates is no problem but mismatching symbol rates has issues. 

For example: My decoder is an stm32f303 with 12-bit ADC at 18k sample rate and 3k symbol rate. If I encode at 48k sample rate and 4k symbol rate (mis-match with decoder) then the decoder will not respond. If I encode at 48k sample rate and 3k symbol rate (matching decoder), everything works fine.

Maybe consider revising the read me to state that "symbol rates on encoder and decoder should match but sample rates need not match. sample_rate must be 6, 8, 12, or 16 times the symbol_rate on both encoder and decoder".

Otherwise, thanks for this great updater. Used it on 700 units with no issues updating in the field. Super solid. About to use it on our next device too.

**Comment by float32** [comment]
I agree, that sentence could definitely use some clarification/correction. Actually the encoder is not as strict as the decoder with the relationship between sample and symbol rates. The sample rate can be any multiple, instead of exactly 6, 8, 12, or 16. So you could encode at 48k with a 1k symbol rate or something like that.

I'm glad you've found it useful, that's great to hear!
